libnopegl/src/backends/vk/rendertarget_vk.c

```diff
@@ -55,22 +55,23 @@ static VkAttachmentStoreOp get_vk_store_op(int store_op)
     return store_op_map[store_op];
 }
 
+#define ARRAY(type, name, size) type name[(size)] = {0}; size_t nb_##name = 0
+
+#define ARRAY_APPEND(name, ...) do { \
+    name[nb_##name] = __VA_ARGS__;   \
```

Comment:
Why a va-arg here? 🤔

Reply:
To allow the following use:
``` c
        ARRAY_APPEND(color_refs, (VkAttachmentReference) {
            .attachment = (uint32_t)nb_descs,
            .layout     = VK_IMAGE_LAYOUT_COLOR_ATTACHMENT_OPTIMAL,
        };
```
Without VA_ARGS you get a compile error as:
``` c
(VkAttachmentReference) {
            .attachment = (uint32_t)nb_descs,
            .layout     = VK_IMAGE_LAYOUT_COLOR_ATTACHMENT_OPTIMAL,
        };
```
is treated as multiple arguments.